nautobot_device_lifecycle_mgmt/templates/nautobot_device_lifecycle_mgmt/softwarenotice_retrieve.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'generic/object_detail.html' %}
```

Comment:
Can this version target Nautobot 2.4 as a minimum and use the UI Component Framework for this view instead of the below HTML?

Reply:
@glennmatthews Unfortunately 3.0 must support 2.2 as a minimum as this is where core introduced Software models that we will start pointing to.